code/modules/antagonists/vampire/vampire_powers/vampire_powers.dm

```diff
@@ -256,9 +256,24 @@
 
 /datum/vampire_passive/vision
 	gain_desc = "Your vampiric vision has improved."
+	var/lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_VISIBLE
+	var/see_in_dark = 1
+	var/vision_flags = SEE_MOBS
+
+/datum/vampire_passive/vision/advanced
+	gain_desc = "Your vampiric vision now allows you to see everything in the dark!"
+	lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_VISIBLE
+	see_in_dark = 5
+	vision_flags = SEE_MOBS
+
+/datum/vampire_passive/vision/full
+	gain_desc = "Your vampiric vision has reached its full strength!"
+	lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE
+	see_in_dark = 8
+	vision_flags = SEE_MOBS
```

Comment:
these see_in_dark values need to be reduced because each value is added rather than just the highest value being added